docs/design/logs.md

```diff
@@ -0,0 +1,240 @@
+# OpenTelemetry Rust Logs Design
+
+Status:
+[Development](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md)
+
+## Overview
+
+OpenTelemetry (OTel) Logs support differs from Metrics and Traces as it does not
+introduce a new logging API for end users. Instead, OTel recommends leveraging
+existing logging libraries such as `log` and `tracing`, while providing bridges
+(appenders) to route logs through OpenTelemetry.
+
+Unlike Traces and Metrics, which introduced new APIs, Logs took a different
+approach due to the long history of existing logging solutions. In Rust, the
+most widely used logging libraries are `log` and `tracing`. OTel Rust maintains
+appenders for these libraries, allowing users to seamlessly integrate with
+OpenTelemetry without changing their existing logging instrumentation.
+
+The `tracing` appender is particularly optimized for performance due to its
+widespread adoption and the fact that `tracing` itself has a bridge from the
+`log` crate. Notably, OpenTelemetry Rust itself is instrumented using `tracing`
+for internal logs. Additionally, when OTel began supporting logging as a signal,
+the `log` crate lacked structured logging support, reinforcing the decision to
+prioritize `tracing`.
+
+## Benefits of OpenTelemetry Logs
+
+- **Unified configuration** across Traces, Metrics, and Logs.
+- **Automatic correlation** with Traces.
+- **Consistent Resource attributes** across signals.
+- **Multiple destinations support**: Logs can continue flowing to existing
+  destinations like stdout while also being sent to an OpenTelemetry-capable
+  backend, typically via an OTLP Exporter or exporters that export to operating
+  system native systems like `Windows ETW` or `Linux user_events`.
+- **Standalone logging support** for applications that use OpenTelemetry as
+  their primary logging mechanism.
+
+## Key Design Principles
+
+- High performance - no locks/contention in the hot path, minimal/no heap
+  allocation.
+- Capped resource usage - well-defined behavior when overloaded.
+- Self-observable.
```

Comment:
Could you clarify this (I assume "the logs subsystem logs about its own work", but not 100%)